mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1331,8 +1343,8 @@ private class LogisticCostFun(
     standardization: Boolean,
     bcFeaturesStd: Broadcast[Array[Double]],
     regParamL2: Double,
-    multinomial: Boolean) extends DiffFunction[BDV[Double]] {
-
+    multinomial: Boolean,
+    depth: Int) extends DiffFunction[BDV[Double]] {
```

Comment:
OK, I will change it to aggregationDepth and create two follow up JIRAs, one for SharedParamsCodeGen.scala and the other about the expert formula.